plugin.go

```diff
@@ -33,6 +34,12 @@ type (
 		Config  map[string]string `json:"config"`
 	}
 
+	InitOptions struct {
+		BackendConfig string `json:"backend_config"`
```

Comment:
I wonder about making those `backend-config`, `lock-state`, etc so that they match TF's command line options...